src/scripts/main.js

```diff
@@ -1,3 +1,301 @@
 'use strict';
 
-// write code here
+// TABLE VARIABLES
+const table = document.querySelector('table');
+const tHead = table.tHead;
+const tBody = table.tBodies[0];
+
+// SORTING FUNCTIONS AND RELATED OBJECTS
+const valueByIndex = (row, index) => row.cells[index].textContent;
+const salaryToNum = (a) => Number(a.substring(1).split(',').join(''));
```

Comment:
It's a good practice to avoid cryptic function names. Consider renaming `valueByIndex` and `salaryToNum` to more descriptive names that indicate what they do. For example, `getValueByCellIndex` and `convertSalaryToNumber`.